docs/getting-started.md

```diff
@@ -1,54 +1,49 @@
+## MDL quick start: building a simple greeting app
 
 The best way to learn any new technology is to get your hands dirty and build something with it, so
-that's what we'll do here! We'll be building a simple web page which lets you enter a first and/or
-last name, and then greets you using your name. You can see a finished example [here](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
+that is what we will do here!  We will be building a simple greeting page which lets you enter a name and greets you as such. Here is the [finished example](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
 
 As you go through this guide, we encourage you to code along with it. By the end, you will have
 learned the fundamentals incorporating MDLv2 into simple sites, as well as worked with some of the
 components we have to offer.
 
 ### Setting up the project
 
-First, let's set up our project. Throughout this guide, we'll assume you have a recent version of
-[NodeJS](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) available on your `$PATH`. You can get the latest NodeJS download [here](https://nodejs.org/en/download/), or use a
-tool like [nvm](https://github.com/creationix/nvm) to install it.
+Before setting up the project, you will need to have a recent version of [NodeJS](https://nodejs.org/). Install it with the [NodeJS installer](https://nodejs.org/en/download/) or through [nvm](https://github.com/creationix/nvm). The [npm](https://www.npmjs.com/) command should made be available on your `$PATH`.
 
-Once node is installed, create a directory for the site, and install MDLv2 inside of it.
+Let's begin creating the project by first creating the directory for the app and add MDLv2. 
+This will install MDLv2 into the `node_modules` folder inside of the `greeting-app` directory.
 
 ```
 mkdir greeting-app
 cd greeting-app
-npm init -y # adds a package.json file into the directory
+npm init -y  # adds a package.json file into the directory
 npm install --save material-design-lite
 ```
 
-This will install MDLv2 into the `node_modules` folder inside of the `greeting-app` directory.
-
-While we're at it, let's also install [live-server](http://tapiov.net/live-server/) so we can
-develop our greeting app using a local server, and have it automatically reload when we make
-changes.
+We recommend installing and using [live-server](http://tapiov.net/live-server/) as the local 
+development server that will automatically reload when changes are made.
 
 ```
 npm install --global live-server
 ```
 
 The `--global` flag tells npm to install the package globally, so that the `live-server` program
-will be available on your path.
+will be available on your `$PATH`.
 
 ### Creating the skeleton index.html file
 
-Now that we have a sane environment set up, let's create a simple `index.html` file, and include
+Now that we have a the environment set up, let us create a simple `index.html` file, and include
```

Comment:
> an environment set up
Just one article.

>, create a simple
Drop the `let us` entirely. Docs should be written in active voice of the second person viewpoint.